app/views/categories/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="jumbotron-category" style='background:
+    linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, 0.2),
+    rgba(0, 0, 0, 0.3)
+  ),
+  url(<%= @category.jumbotron_photo_url %>) no-repeat center center;
+background-position:
+<% if @category.name == "Seashells" %>
+  50% 80%;'>
+<% elsif @category.name == "Clothing & Accessories" %>
+  20% 90%;'>
+<% else %>
+  50% 20%;'>
+<% end %>
+  <h2><%= @category.name.upcase %></h2>
+</div>
+<div class="container container-home-page">
+  <%= render 'layouts/navbar' %>
+  <% @category.products.each_with_index do |product, i| %>
+  <% if product.status == "Active" && product.quantity != 0 %>
+      <% if i + 1 % 3 == 0 %>
```

Comment:
See my note on line 6 in views/products/all_products.html.erb about using % to control visual display. It can lead to unexpected behavior (e.g., b/c not all the visual cards are the same size), so I'd use bootstraps cols instead.